### PR TITLE
Added Jupyter Notebook for Overview of OpenCV

### PR DESCRIPTION
**Pull request by dhairyashah1**

### CODE REVIEW

File: cv-basics/OpenCV_Overview.ipynb [hunk 1]
```diff
@@ -0,0 +1,939 @@
+{
```
Comment (gautam-dev-maker) [inline cv-basics/OpenCV_Overview.ipynb:1]: <p>put matplotlib content under different heading.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SRA-VJTI/Intro_to_CV/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (gautam-dev-maker) [inline cv-basics/OpenCV_Overview.ipynb:1]: <p>Add pictorial depiction , wherever image has been processed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SRA-VJTI/Intro_to_CV/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (amanchhaparia) [inline cv-basics/OpenCV_Overview.ipynb:1]: <p>Please read the documentation of cv2.waitkey() 0 means forever</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SRA-VJTI/Intro_to_CV/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (dhairyashah1) [reply cv-basics/OpenCV_Overview.ipynb:1]: added

Comment (dhairyashah1) [reply cv-basics/OpenCV_Overview.ipynb:1]: added samples of pictorial descriptions

Comment (dhairyashah1) [reply cv-basics/OpenCV_Overview.ipynb:1]: change encorporated, thanks

**Comment by amanchhaparia** [comment]
Add a table on top describing the function name and its description

**Comment by dhairyashah1** [comment]
@amanchhaparia  tables are added for functions used in OpenCV. for details they may refer to the Documentation.

**Comment by amanchhaparia** [comment]
> @amanchhaparia tables are added for functions used in OpenCV. for details they may refer to the Documentation.

Thnks for the contribution

**Pull request merged**